region/norfair/east/Norfair Reserve Tank Room.json

```diff
@@ -78,6 +78,11 @@
       "id": "A",
       "name": "Lava Dragons",
       "obstacleType": "enemies"
+    },
+    {
+      "id": "B",
+      "name": "Sovas",
+      "obstacleType": "enemies"
```

Comment:
There are a few more things we probably need to fix up if this moves forwards... a couple of strats that will kill these guys, and we need to mark strats like `Frozen Sova Platform` (27) with `obstaclesNotCleared: B`.  Unsure whether the `canTrickyUseFrozenEnemies` in strat 24 is with the Sova or with the Dragon.